Source/msg.cpp

```diff
@@ -187,7 +187,7 @@ std::string_view CmdIdString(_cmd_id cmd)
 #endif // LOG_RECEIVED_MESSAGES
 
 struct TMegaPkt {
-	uint32_t spaceLeft;
+	size_t spaceLeft;

```

Comment:
We never serialize this directly, right? Cause if we did that'd be a 32/64-bit network incompatibility.

Reply:
This is correct. `TMegaPkt` is never serialized. That's also why it has no `#pragma pack` around it.